cfg/osic1p.cfg

```diff
@@ -2,24 +2,28 @@ FEATURES {
     STARTADDRESS: default = $0200;
 }
 SYMBOLS {
+# Add "-u __BOOT__" to a command line if you want a file that can be loaded directly.
+#    __BOOT__:      type = import;
```

Comment:
>> If they never use C1Pjs, then they simply could uncomment the lines in the two configure files; they wouldn't need to type a full line into those files.
>
> The above was, in fact, clear to me. But, I think it should be mentioned in a comment. Maybe it would make much more sense to tell that in a comment than the '-u ...' hint.

Good point.
Is the new comment good enough?